src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -82,9 +82,10 @@ void handleSharedPythonMonkeyMemory(JSContext *cx, JSGCStatus status, JS::GCReas
   if (status == JSGCStatus::JSGC_BEGIN) {
     PyToGCIterator pyIt = PyTypeToGCThing.begin();
     while (pyIt != PyTypeToGCThing.end()) {
+      PyObject *pyObj = pyIt->first->getPyObject();
       // If the PyObject reference count is exactly 1, then the only reference to the object is the one
       // we are holding, which means the object is ready to be free'd.
-      if (PyObject_GC_IsFinalized(pyIt->first->getPyObject()) || pyIt->first->getPyObject()->ob_refcnt == 1) {
+      if (_PyGC_FINALIZED(pyObj) || pyObj->ob_refcnt == 1) { // PyObject_GC_IsFinalized is only available in Python 3.9+
```

Comment:
On Python 3.9+, `_PyGC_FINALIZED` is simply an alias of `PyObject_GC_IsFinalized`
https://github.com/python/cpython/blob/v3.9.17/Include/cpython/objimpl.h#L131 
```c++
#  define _PyGC_FINALIZED(o) PyObject_GC_IsFinalized(o)
```

Reply:
there is still also `Py_SET_SIZE`, `PyObject_CallOneArg` (which I believe is aliased to `_PyObject_CallOneArg`?), and `PyObject_CallNoArgs`, and potentially other function calls that we may decide to use in the future.